contracts/wallet.sol

```diff
@@ -261,18 +261,18 @@ library DailyLimitTrait {
     /// @param _amount is the daily limit amount in base units.
     function _setLimit(DailyLimit storage self, uint _amount) internal {
         // Require that the spend limit has not been set yet.
-        require(!self.set, "daily limit has already been set");
+        require(!self.updateable, "daily limit has already been set");
```

Comment:
maybe we should change the require string to "updated" or "limit not updateable"